beginnercodes/challenges/test_runner.py

```diff
@@ -23,17 +29,41 @@ class Result:
     status: StatusEnum = field(default=StatusEnum.SUCCESS)
 
 
-def test(challenge: int, solution_func):
+def build_table():
+    _t = Table(title='[bold]Test Results[/bold]', highlight=True, box=box.MINIMAL)
+    _t.add_column("#:", justify='right')
+    _t.add_column("Expected:", justify='center')
+    _t.add_column("Got:")
+    return _t
+
+
+def test(challenge: int, solution_func, description: bool = False, examples: bool = False):
```

Comment:
Make `solution_func` optional, no need to pass in a function if you just wanna get the description or examples